sklearn/datasets/_svmlight_format.pyx

```diff
@@ -64,7 +64,10 @@ def _load_svmlight_file(f, dtype, bint multilabel, bint zero_based,
 
         target, features = line_parts[0], line_parts[1:]
```

Comment:
I just checked the code for the suggestions of @jnothman . The current code handles the "no-active feature" cases correctly. In particular,

(0) no-active feature and no label: handled by the "if len(line_parts) == 0"
(1) no-active feature and some label(s): handled normally by having target = line_parts[0] and features = line_parts[1:] (which would just be []).

So IMHO no other special handling is needed. Thanks.